### DISCUSSION TITLE
Can't update from 106.3 to 107.2

### DISCUSSION THREAD

**Discussion by icolaice** [AdguardTeam/AdGuardHome#4163] [Q&A]
路由器上运行的AGH106.3手动可以检测到更新，但仅仅提示：检查更新成功，却不能更新。之前手动检查到更新后，AGH会自动更新的，不知道为什么现在不行了。不知道如何解决，请求帮助。

**Comment by ameshkov** [comment]
我们需要看看AGH的日志。

1. [Configure](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog) AdGuard Home to write verbose-level log.
2. Reproduce the issue.
3. Post the log file here.

**Comment by lucktu** [comment]
> 
自己下载后面的插件，解压以后，将里面的主程序换掉即可（换之前停止adg）。
